templates/roundPicker.html

```diff
@@ -10,7 +10,7 @@
                  class="brand"
                  data-toggle="tooltip" data-placement="bottom" data-container="body"
                  title="{{eventName eventCode}}">
-                <img class="img-lg" src="{{eventIcon eventCode}}" />
+                <span class="cubing-icon icon-{{eventCode}}"></span>
```

Comment:
We're inside of an anchor tag with a tootlip here, so I'm not going to bother with an alt tag.